training/pulid_flux_images.py

```
@@ -8,13 +8,14 @@
+# ScaleX only handles upscaling (fal-ai/esrgan), not image generation
+# PuLID Flux (fal-ai/flux-pulid) still needs original fal_client
 try:
-    import scalex_client as fal_client
-    print("✅ Using ScaleX-CLI for PuLID Flux processing (replacing FAL AI)")
-except ImportError:
-    print("⚠️ ScaleX-CLI not available, falling back to original fal_client")
     import fal_client
+    print("✅ Using FAL AI for PuLID Flux image generation")
+except ImportError:
+    print("❌ fal_client not available - PuLID Flux requires FAL AI")
+    raise ImportError("PuLID Flux requires fal_client for image generation")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)